src/Resources/administration/module/custom-module/snippet/en-GB.json

```diff
@@ -0,0 +1,4 @@
+{
+    "demoSnippet": "I'm a demo."
+
```

Comment:
Unnecessary line

Reply:
So you'd prefere an empty example-file? ok

Reply:
No. There was an unnecessary empty line.
```
{
    "demoSnippet": "I'm a demo." // Good line
    // Bad line
}
```

Reply:
Oh :thinking: